xmtp_api_grpc/src/grpc_api_helper.rs

```diff
@@ -33,7 +33,6 @@ async fn create_tls_channel(address: String) -> Result<Channel, Error> {
         .map_err(|e| Error::new(ErrorKind::SetupError).with(e))?
         .keep_alive_while_idle(true)
         .connect_timeout(Duration::from_secs(5))
-        .http2_keep_alive_interval(Duration::from_secs(10))
```

Comment:
I bumped this all the way down to 1 and it still wouldn't pass. Removing it seems to be the only way to make it pass. Do we need this?

Reply:
I thought the default of 20 seconds would kick in if we removed this setting, which might break this some time later than 50 seconds. 

Maybe I'm wrong about the defaults

Reply:
I set it to 300 seconds and it still didn't break. I can try even longer.

Reply:
Okay just set the delay to 1000 seconds which is like 16 minutes. And it still passed. So I think this is safe.

Reply:
There's a risk that this is going to break the disconnect detection on mobile, but that feels less bad than not streaming at all.